agent/agent_test.go

```diff
@@ -1,26 +1,38 @@
 package agent
 
 import (
+	"errors"
+	"fmt"
+	"net"
+	"sync"
 	"testing"
 	"time"
 
+	"google.golang.org/grpc"
+
 	"github.com/docker/swarmkit/agent/exec"
 	"github.com/docker/swarmkit/api"
 	"github.com/docker/swarmkit/ca"
 	"github.com/docker/swarmkit/ca/testutils"
 	"github.com/docker/swarmkit/connectionbroker"
+	raftutils "github.com/docker/swarmkit/manager/state/raft/testutils"
```

Comment:
Non-blocking: at some point we should reorganize packages so that `PollFuncWithTimeout` is not in `raftutils`.